site/src/testHelpers/entities.ts

```diff
@@ -826,7 +826,8 @@ export const MockTemplate: TypesGen.Template = {
 	deprecated: false,
 	deprecation_message: "",
 	max_port_share_level: "public",
-	use_classic_parameter_flow: false,
+	use_classic_parameter_flow: true,
```

Comment:
why does this need to be true?

the classic flow will be removed soon, so it's important that new functionality work with the new flow